backend/src/app.module.ts

```diff
@@ -1,15 +1,15 @@
-import { Module } from '@nestjs/common';
+import {forwardRef, Module} from '@nestjs/common';
```

Comment:
let's use eslint and prettier, can you implement it?

Reply:
thanks , done